jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGenerator.java

```diff
@@ -99,6 +99,35 @@ private static void addIfNotEmpty(
             layerEntry.getSourceFiles(), directoryInContext, layerEntry.getExtractionPath()));
   }
 
+  /**
+   * Converts a map to a corresponding dockerfile string in the form of:
+   *
+   * <pre>{@code
+   * command key1="value1" \
+   *     key2="value2" \
```

Comment:
nit: should we line up `key1` and `key2` since we're adding spaces anyways?

Reply:
The command names are different lengths, and adding a variable number of spaces here would have harmed readability or required a weird extra parameter, so I don't think it's worth it, unless there's a function we can use to generate whitespace that I don't know about (that doesn't require another `import`). I think it's fine as is.